wiki/Tournaments/Badge-weighted_seeding/de.md

```diff
@@ -0,0 +1,61 @@
+
+Das Prinzip hinter BWS ist, dass Spieler eine passendere Einstufung erhalten, wenn sie in der Vergangenheit Abzeichen für die Teilnahme an Turnieren erworben haben. Dadurch können Spieler im Vergleich zu ähnlich eingestuften Spielern nicht unbegrenzt auf demselben Platz bleiben, wenn sie großartige Erfolge erzielen.[^digitalhypno-discord-1][^digitalhypno-discord-2] Beim Vergleich mehrerer Turniere, die alle BWS verwenden, wird erwartet, dass Spieler auf zu Beginn vergleichbaren Rängen zu BWS-Seeds tendieren, die ihre Wahrscheinlichkeit, Spiele gegen ähnlich platzierte Spieler zu gewinnen, besser reflektiert.[^digitalhypno-discord-3][^digitalhypno-discord-4]
+
+BWS wurde erstmals Ende 2018 beim [Hippo Cup 2](https://osu.ppy.sh/community/forums/topics/848153) eingesetzt. Seitdem wird es in vielen [Community-Turnieren](/wiki/Tournaments#community) verwendet und ist neben den globalen Rängen und den Qualifikationsrunden eine der wichtigsten Auslosungsverfahren, die derzeit im Einsatz sind.
+
+## Technischer Hintergrund
+
+Die ursprüngliche und gebräuchlichste Methode zur Berechnung des Spieler-Seeds in BWS ist die folgende beschränkte Exponentialfunktion:
+
+```
+Seed = Globaler Rang ^ (0.9937 ^ (Anzahl Abzeichen ^ 2))
+```
+
+- `Anzahl Abzeichen` (≥ 0): die Anzahl an Abzeichen auf dem Profil eines Spielers, die als Preise für vergangene Turniere desselben [Spielmodus](/wiki/Game_mode) gewonnen wurden
+- `Globaler Rang` (≥ 1): der [Performance-Punkte-Rang](/wiki/Ranking#performance-points-ranking) auf der globalen Rangliste gemäß dem [Spielmodus](/wiki/Game_mode) des Turniers
+
+Diese Funktion erfüllt Eigenschaften, die für den Zweck von BWS wesentlich sind:
+
+- Der Seed eines Spielers sinkt in größeren Schritten, als die Spieler weitere Abzeichen erlangen können, oder mit anderen Worten, jedes neue Abzeichen ist mehr Wert als das vorherige.[^hippo-cup-bws] Beachte, dass dies in der obigen Formel nur bis zu einem Punkt zwischen 2 und 6 Abzeichen zutrifft (abhängig von `Globaler Rang`). Danach tritt ein Wendepunkt ein und der Effekt kehrt sich ins Gegenteil um.
+- BWS-Seeds sind immer kleiner als oder gleich `Globaler Rang`. Sie sind gleich, wenn der Spieler keine Abzeichen besitzt.
+- BWS-Seeds haben dieselbe Größenordnung wie `Globaler Rang`.
```

Comment:
I prefer the current version, though here are the alternatives I had noted:

- `Verteilung`
- `Spektrum`
- `Bereich`

Reply:
seems fine to me